rest/data/RestStore.js

```diff
@@ -20,16 +20,22 @@ export class RestStore extends UrlStore {
 
     _lookupsLoaded = false;
 
-    constructor({dataRoot = 'data', ...rest}) {
+    /**
+     * @param {string} [dataRoot] - Name of root node for records in returned data
+     * @param {boolean} [eagerLoadLookups] - Whether lookups should be loaded each time loadAsync is called
```

Comment:
how about  'reloadLookupsOnLoad'?  Seems slightly clearer, and gets to the point of *re*loading.